### DISCUSSION TITLE
eslint, date-time comment & max-len

### DISCUSSION THREAD

**Discussion by dan-klasson** [dotansimha/graphql-code-generator#4314] [General]
This particular comment in my Scalar interface makes my eslint fail:

`  /** A date-time string at UTC, such as 2019-12-03T09:54:33Z, compliant with the date-time format. */ `

Prettier nor eslint fixes comments that are longer than specified length. Thus adding eslint or prettier hooks in `codegen.yml` doesn't fix the problem.

How to get around this issue?

**Comment by dotansimha** [comment]
@dan-klasson it feels like this should be an issue and not a discussion. If it's still relevant, please report this as issue, with a reproduction.
